auth/api/http/keys/endpoint_test.go

```diff
@@ -144,14 +144,14 @@ func TestIssue(t *testing.T) {
 			req:    toJSON(ak),
 			ct:     contentType,
 			token:  "wrong",
-			status: http.StatusForbidden,
+			status: http.StatusUnauthorized,
```

Comment:
Please fix the description here: `issue key unauthorized` to `issue key with an invalid token`. Also, check other places in tests for the same problem.